Specs/NumberWithUnit/Dutch/TemperatureModel.json

```diff
@@ -420,7 +420,7 @@
     "NotSupportedByDesign": "javascript,python,java"
   },
   {
-    "Input": "converteer 200 graden Celsius celsius naar Fahrenheit",
+    "Input": "converteer 200 graden Celsius naar Fahrenheit",
     "Results": [
       {
```

Comment:
They also need to be removed from the expected output, otherwise tests won't pass, like currently.
@LionbridgeCS2 @LionbridgeCSII

Reply:
@tellarin, fixed.